src/payment/klarna-setup.md

```diff
@@ -31,96 +31,91 @@ _Klarna Merchant Portal_
+   ![Klarna Payments]({% link images/images/config-sales-payment-methods-klarna-payments.png %}){: .zoom}
+   [_Klarna Payments_]({% link configuration/sales/klarna.md %})
 
-   - If your Klarna account supports B2B payments, set **Enable B2B** to `Yes`.
-   - If you want Klarna to prepare a set of personalized credit options for each customer, set **Enable Data sharing** to `Yes`.
+1. Set **Enable** to `Yes`.
 
-   - Set **Enable Data sharing on load** to one of the following:
+1. Set **Allowed Countries** to one of the following:
 
-     |**No** |Customer billing and shipping address information is shared with Klarna after the transaction is authorized. If a shipping address error is encountered during authorization, the customer is instructed to return to the previous step and correct the problem.|
-     |**Yes** |Customer billing and shipping address information is shared with Klarna when the payment method is selected during the checkout process.|
+   |**All Allowed Countries** |Customers from all [countries]({% link stores/country-options.md %}) specified in your store configuration can use this payment method.|
+   |**Specific Countries** |When selected, the Payment from Specific Countries list appears. Hold down the Ctrl key (PC) or the Command key (Mac) and select each country in the list where customers can make purchases from your store.|
 
-   - Enter a **Sort Order** number to determine the position of Klarna in the list of payment methods during checkout. (0 = first, 1 = second, 2 = third, and so on.)
+1. If your Klarna account supports B2B payments, set **Enable B2B** to `Yes`.
 
-     ![Klarna Payments]({% link images/images/config-sales-payment-methods-klarna-payments.png %}){: .zoom}
-     [_Klarna Payments_]({% link configuration/sales/klarna.md %})
+1. If you want Klarna to prepare a set of personalized credit options for each customer, set **Enable Data sharing** to `Yes`. Then, set **Enable Data sharing on load** to one of the following:
 
-1. If you want to stop here, tap <span class="btn">Save Config</span>.
+   |**No** |Customer billing and shipping address information is shared with Klarna after the transaction is authorized. If a shipping address error is encountered during authorization, the customer is instructed to return to the previous step and correct the problem.|
+   |**Yes** |Customer billing and shipping address information is shared with Klarna when the payment method is selected during the checkout process.|
 
-1. When prompted, click the [Cache Management]({% link system/cache-management.md %}) link at the top of the page.  Then, refresh all invalid caches.
+1. Enter a **Sort Order** number to determine the position of Klarna in the list of payment methods during checkout. (0 = first, 1 = second, 2 = third, and so on.)
 
 ## Step 3: Complete the Design Settings **(Optional)**
```

Comment:
i don't think i've seen this.  bold on a header?  isn't the header already bold?  maybe ital?

Reply:
I don't think it needs either -- the parens are enough. Thanks!